manifests/plugin/modbus.pp

```diff
@@ -0,0 +1,42 @@
+# @summary Install and configure the modbus plugin
+#
+# @see https://collectd.org/wiki/index.php/Plugin:Modbus
+#
+# @param ensure Enable/Disable modbus support
+# @param manage_package Install collectd-modbus package?
+# @param data modbus data entries
+# @param hosts modbus host entries
+class collectd::plugin::modbus (
+  Enum['present', 'absent']               $ensure         = 'present',
+  Optional[Boolean]                       $manage_package = undef,
+  Hash[String[1], Collectd::Modbus::Data] $data           = {},
+  Hash[String[1], Collectd::Modbus::Host] $hosts          = {},
+) {
+  include collectd
+
+  $_manage_package = pick($manage_package, $collectd::manage_package)
+
+  if $facts['os']['family'] == 'RedHat' {
```

Comment:
These two if statements are very duplicated.

Maybe just set a $_pkgname via a selector and use tha.